pwiz_tools/Skyline/Model/Results/IonMobilityFinder.cs

```diff
@@ -258,8 +258,8 @@ private bool ProcessFile(ChromFileInfo fileInfo)
             // Only use the transitions currently enabled
             var transitionPointSets = chromInfo.TransitionPointSets.Where(
                 tp => nodeGroup.Transitions.Any(
-                    t => (t.Mz - (tp.ExtractionWidth ?? tolerance)/2) <= tp.ProductMz &&
-                         (t.Mz + (tp.ExtractionWidth ?? tolerance)/2) >= tp.ProductMz))
+                    t => (t.Mz - (tp.ExtractionWidth ?? tolerance.Interval) / 2) <= tp.ProductMz &&

```

Comment:
This "Tolerance.Interval" does not seem to be the right thing to be using if the tolerance is ppm.

Reply:
Agree.